newm-server/src/main/kotlin/io/newm/server/features/arweave/repo/ArweaveRepositoryImpl.kt

```diff
@@ -85,6 +90,9 @@ class ArweaveRepositoryImpl(
         )
     }
     private val arweaveConfig by lazy { configApi.apply(arweaveUri, arweaveBackend) }
+    private val minWalletBalance: BigDecimal by lazy {
+        environment.getConfigDouble("arweave.minWalletBalance").toBigDecimal()
```

Comment:
Go from string to big decimal. If you use double, precision can be lost.

Reply:
Done